### ISSUE TITLE
Request for ability to clear Warehouse caches

### ISSUE THREAD

**Issue by Gary-van-Breda** [Indicia-Team/warehouse#149]
Can I make a request to add a mechanism to clear at least some of the warehouse caches (similar to Drupal iform functionality) - I wouldn't touch the authentication ones. There are times (like changing the required state of an attribute) where we have to wait an arbitrary time before the cache is redone, and we have no ability to speed the process.

**Comment by johnvanbreda** [comment]
Agreed, this would be a good idea.

**Comment by dve** [comment]
I agree that it would be a good option to clean caches manually. But if there is an issue with caching and the required state of an attribute that should be fixed separate.
